### ISSUE TITLE
more explaination needed

### ISSUE THREAD

**Issue by touseefbsb** [Team-FLUI/FluiFramework#1]
Hey there, I like all of your projects and all work you do in uwp community, just wanted to suggest that maybe its better to put some more examples in readme and explain a little how this library can be productive for uwp devs, also how will this work with winui library since that is currently the standard fluent design controls library which automatically adapts to windows version.
Having said that I wanna say this does looks promising, and I appreciate the work you do in uwp :)

**Comment by ikarago** [comment]
Thanks for the comment! I've updated the Readme a bit, I hope the usage and the purpose of the package is a little bit more clear now. :) If so, feel free to close the Issue, and if not, please let me know what you're still missing. ;)

**Comment by touseefbsb** [comment]
yeah so readme is improved and it helps thanks however I have some questions.

1. so these set of styles you have I assume you copied them from generic.xaml from a specific sdk version? and then made them fluent and that is wht we will be using? Assuming that is the case, the worry I have with it is it will be static styles and for example new templates for controls come in new sdk version or winui itself we wont be able to take advantage of those styles automatically assuming our app is running on multiple windows 10 versions. So what plan do you have to encounter that situation? I think that can be solved by including this project to winui itself? so that all controls are updated regularly and work across different versions?

Apart from that worry I really like this project and the idea seeing we can do fluent style on controls like textblock, texbox etc bcz these controls were lacking it in winui library.
